src/test/java/com/github/reducktion/socrates/validator/GermanyIdValidatorTest.java

```diff
@@ -0,0 +1,107 @@
+package com.github.reducktion.socrates.validator;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class GermanyIdValidatorTest {
+    private GermanyIdValidator germanyIdValidator;
+
+    @BeforeEach
+    void setup() {
+        germanyIdValidator = new GermanyIdValidator();
+    }
+
+    /**
+     * These valid numbers are taken from the official document,
+     * mentioned in the JavaDoc of {@link GermanyIdValidator#validate}.
+     *
+     * @param validId an ID to test
+     */
+    @ParameterizedTest
+    @ValueSource(strings = {
+        "86095742719",
+        "47036892816",
+        "65929970489",
+        "57549285017",
+        "25768131411"
+    })
+    void validate_shouldReturnTrue_whenIdMatchesDocumentedOne(final String validId) {
+        assertThat(germanyIdValidator.validate(validId), is(true));
+    }
+
+    @Test
+    void validate_shouldReturnFalse_whenLeadingZeroIndicatesaTestIdentifierAndIsNotAllowed() {
```

Comment:
I don't see why this simplifies ... but that's just a matter of taste.
Fair for me anyway, since I trust in your overview of the project and aim for consistency.